pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -606,12 +625,22 @@ def _get_version(self):
             return _extract_version('')
         else:
             # specify logging to stdout for windows compatibility
-            # technically this command makes gams complain because we're not
-            # providing a filename, but it will include the version name anyway
-            cmd = [solver_exec, "", "lo=3"]
+            cmd = [solver_exec, "audit", "lo=3"]
             _, txt = pyutilib.subprocess.run(cmd, tee=False)
             return _extract_version(txt)
 
+    @staticmethod
+    def _parse_special_values(value):
+        if value == 1.0e300 or value == 2.0e300:
+            return float('nan')
+        if value == 3.0e300:
+            return float('inf')
+        if value == 4.0e300:
+            return -float('inf')
+        if value == 5.0e300:
+            return sys.float_info.epsilon
+        return value
```

Comment:
out of curiosity, are these documented somewhere? I wasn't able to find them doing a cursory scan of GAMS documentation.

Reply:
Not sure. Usually, those values are transformed in most interfaces except the 'expert-level' interfaces (I guess). So a user wouldn't normally see them. We should improve the documentation of the 'expert-level' APIs though...